### DISCUSSION TITLE
[Weekly Review Report]: February 19 - 23

### DISCUSSION THREAD

**Discussion by tabbassidaloii** [wikipathways/wikipathways-help#108] [curation-reports]
### Recent changes

* many pathways were edited by Eweitz (e.g. ontology terms were added, typos fixed, soften disease color, standardize case)
* [WP3878](https://classic.wikipathways.org/index.php/Pathway:WP3878): There is a link from Chromatin retention that is not clear to which node it connects.

### Under construction

 * [WP5432](https://classic.wikipathways.org/index.php/Pathway:WP5432): some nodes to be annotated, discription to be added 
 

### New pathways

No new pathways

### Deleted Pathways

 * deletion requested for [WP2512](https://classic.wikipathways.org/index.php/Pathway:WP2512) by [Could not find out]. I agree with this request. 

### Update Approved pathway tag

- [X] All Approved pathways tags were updated!

**Comment by khanspers** [comment]
Thanks @tabbassidaloii! I have updated [WP3878](https://classic.wikipathways.org/index.php/Pathway:WP3878) to better indicate the connection from the MRE11 complex. The exact connection with the cell cycle checkpoint is not clear from the paper, so I updated it to more clearly show a "cell cycle checkpoint" group, with the interaction from MRE11.

I have deleted [WP2512](https://classic.wikipathways.org/index.php/Pathway:WP2512).
